sgt-back-end/index.js

```diff
@@ -0,0 +1,130 @@
+const express = require('express');
+const app = express();
+const pg = require('pg');
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const jsonMiddleWare = express.json();
+
+app.use(jsonMiddleWare);
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+  select *
+    from grades
+    `;
+  db.query(sql)
+    .then(result => {
+      const grades = result.rows;
+      res.status(200).json(grades);
+    })
+    .catch(error => {
+      // eslint-disable-next-line no-console
+      console.log(error);
+      res.status(500).json({ error: 'An unexpected error occurred.' });
+    });
+});
+
+app.post('/api/grades', (req, res) => {
+  const body = req.body;
+  const score = Number(req.body.score);
+  if (!body.name || !body.course || !body.score) {
+    res.status(400).json({ error: 'Entry must contain "Name", "Course", and "Score"' });
+    return;
+  } else if (!score || score < 0 || score > 100 || !Number.isInteger(score)) {
```

Comment:
`0` should be a valid `score`:

---

![image](https://user-images.githubusercontent.com/16143771/151639027-f24e5789-dcc4-499c-ae06-008cee7da981.png)